content/general/concepts/big-data/big-data.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: 'Big Data'
+Description: 'Big data involves working with and developing insights from large datasets.'
+Subjects:
+  - 'Data Science'
+  - 'Data Visualization'
+Tags:
+  - 'Database'
+  - 'Queries'
+CatalogContent:
+  - 'r-for-programmers'
+  - 'paths/data-science'
+---
+
+<link rel="canonical" href="https://www.codecademy.com/resources/blog/what-is-big-data/" />
+
+Big data involves working with and developing insights from large datasets.
+
+The key distinctions between regular data and big data are volume, velocity, and variety. Generally, big data is more extensive information with more individual components that are collected in a shorter period of time. Big data sources are often new but can encompass older data streams as well.
+
+## Examples
+
+Big data is utilized across multiple industries and applied in many ways, including the following:
+
+- Marketing departments use this data for targeted advertising, promoting products, and services that align with company interests.
+- Healthcare professionals may use big data gathered from devices like wearables and track metrics such as heart rate, breathing, and sleep habits.
+- The transportation and automobile industries may use big data to track informations related to pedestrians, weather, and road conditions.
```

Comment:
The first bullet point here does a good job of explaining what big data is used for in marketing. Maybe we can include a short sentence about what the collected info for healthcare + transportation is ultimately used for.

In healthcare, this info can be used for public health surveillance and helping people with disabilities (exact terminology) perform ADLs (activities of daily living). 

In transportation, this info is used every day for improving road safety, and response and preventative weather measures.